test/Tests/Traits/UsesInputFiles.php

```diff
@@ -15,13 +15,15 @@ public function getInputPath(string $appends = ''): string
         return $path;
     }
 
-    public function getInputFileContents(string $path): string
+    public function getInputFileContents(string $path, bool $addEOL = false): string
```

Comment:
idk why but sometimes when comparing output from action we need extra EOL this just seemed easy way to do it.